pkg/app/app.go

```diff
@@ -75,6 +78,33 @@ func RealMain(opts types.Options, d types.Deployer, tester types.Tester) (result
 		return errors.Wrap(err, "could not create runner output")
 	}
 	writer := metadata.NewWriter("kubetest2", junitRunner)
+
+	var wg sync.WaitGroup
+	wg.Add(1)
+	done := make(chan bool)
```

Comment:
we can defer a done here to avoid needing to do it explicitly in every exit case

`defer func(){ done <- true }()`